engine/src/main/java/org/terasology/rendering/dag/nodes/OutlineNode.java

```diff
@@ -75,23 +75,13 @@ public void initialise() {
      */
     @Override
     public void process() {
-        if (renderingConfig.isOutline()) { // TODO: define state changes after this check is eliminated
-            PerformanceMonitor.startActivity("rendering/outline");
-            outlineFBO = displayResolutionDependentFBOs.get(OUTLINE);
-
-            outline.enable();
-
-            // TODO: verify inputs: shouldn't there be a texture binding here?
-            outlineFBO.bind();
+        PerformanceMonitor.startActivity("rendering/outline");
 
-            setViewportToSizeOf(outlineFBO);
-            glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT); // TODO: verify this is necessary
+        glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT); // TODO: verify this is necessary
+        renderFullscreenQuad();
+        bindDisplay();  // TODO: verify this is necessary
```

Comment:
I'm pretty sure we can remove this now. If necessary the default state change task will do it. Just verify enabling/disabling the outline still works ok.